platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,482 @@
+
+###### Characteristics:
+
+* It becomes a priority to enable specialists to extend platform capabilities and introduce new ones
+* The organization can centralize specialists allowing their knowledge and support to be spread through platform capabilities
+
+###### Examples Scenarios:
+
+* Marketing works with platform builders to introduce consistent user tracking in order to attribute marketing efforts to product outcomes. 
+* Automation initiative reduces human time to provision databases by 30 minutes per instance, saving $10m/year
+
+
+### Adoption
+<h4 style="color:gray"><i>Why and how do users discover and use internal platforms and platform capabilities?</i></h4>
+
+Adoption describes not only how and how much an organization uses platform capabilities, but also what motivates them to do so. In the early stages,  many target users may not realize they are using a platform at all, rather they see their tools as an ad hoc collection of capabilities from various internal sources. This may mature into a group of capabilities that is consistently managed and presented to users - that is, one or more platforms. As the capabilities become more refined and discoverable, it is common that the drive for platform use moves away from more external motivations like mandates or incentives. Allowing users to self-select into platform capabilities and ideally even invest their own efforts into the wider platform ecosystem.
+
+
+<p align="center">
+<img src="assets/adoption-curve.jpg" width=600px />
+<br/>
+<em>A diagram to indicate a common growth pattern for platform</br>adoption. This showcases the often slow start driven mainly by platform builders.</br></br>Once platforms provides enough value to users, growth becomes more pulled by</br>the users causing a steeper adoption curve.</em>
+</p>
+
+
+
+##### Level 1, Provisional - Erratic
+
+Adoption of shared platforms and capabilities is sporadic and inconsistent. No organization-wide strategy or guidance exists for choosing and integrating required backing services and technologies. Individual teams might leverage platform and DevOps practices to improve their own processes, but there is no coordinated effort or standardization across the organization. This level of adoption is characterized by the absence of a coherent approach and the idea that external tools are more effective than those provided internally.
+
+###### Characteristics:
+
+* One-off tools, services, and capabilities are managed by and consumed by from various teams and departments in the organization
```

Comment:
```suggestion
* One-off tools, services, and capabilities are managed by and consumed by various teams and departments in the organization
```
Might even be possible to remove the first "by", however, it might help clarity even more to expand the sentence to make clear that manager and consumer might be different groups. Maybe someone with deeper copy editing prowess can suggest better phrasing.